apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue

```diff
@@ -245,6 +253,13 @@ const handleDynamicLayoutFetch = (changed) => {
 const handleVisibleCustomFieldModal = (visible) => {
     tableState.visibleCustomFieldModal = visible;
 };
+const handleCloseAddClusterModal = () => {
+    router.replace({
+        name: ASSET_INVENTORY_ROUTE.SERVICE_ACCOUNT._NAME,
+        query: { provider: 'kubernetes' },
+    });
```

Comment:
question: Don't you have to use getProper...()?

Reply:
😨